h2/src/main/org/h2/mvstore/MVMap.java

```diff
@@ -1705,34 +1753,30 @@ public void reset() {}
 
     public V operate(K key, V value, DecisionMaker<? super V> decisionMaker) {
         beforeWrite();
+        IntValueHolder unsavedMemoryHolder = new IntValueHolder();
```

Comment:
`IntValueHolder` needs an explicit constructor with default visibility to avoid synthetic access here and in another place.